src/Chapter3.hs

```diff
@@ -1048,11 +1063,12 @@ isWeekend Sun = True
 isWeekend _ = False
 
 nextDay :: Weekday -> Weekday
-nextDay = succ
+nextDay Sun = Mon
```

Comment:
That is right!
You could also use the instance of `Bounded` in here:

```suggestion
    if day == maxBound then minBound else ...
```

This is also a bit more maintainable. If you decide, for example, to have the "American" system of week days in the data type constructors, you won't need to change this function at all! 🙂